hw/dv/data/common_sim_cfg.hjson

```diff
@@ -24,7 +24,7 @@
   // pass and fail patterns
   build_pass_patterns: []
   build_fail_patterns: ["^ERROR:.*$"] // fusesoc build error
-  run_pass_patterns:   ["^TEST PASSED (UVM_)?CHECKS$"]
+  run_pass_patterns:   ["TEST PASSED (UVM_)?CHECKS$"]
```

Comment:
Is this change (removal of ^) needed as well?

Reply:
Ohh, I just saw your comment. Regardless of the simulator used, the signature should always be printed on a newline. Is it not working for you? UVM_* messages that we print from our code do not start at the start of the line, so this makes it less likely to accidentally replicate.

Reply:
Some tools adds prefix at the start of line in this case Riviera adds "KERNEL:" to show from which sub-tool message was printed. 
If this is an issue we should filter this prefix in the log file before is is checked for error. It can be done using sed/awk on stream or on log file.  Could you point best place for such operation.
Anyway I think that changing reg exp so it do not look for pattern from start fo the line should be ok and it is better  solution here than filtering.

Reply:
Or I can add to file `riviera_run.do`:
```
unset  messageprefix
pref.setvalue application/console/general/show-identifier-in-messages false
run -all
endsim
pref.setvalue application/console/general/show-identifier-in-messages true
```
@sriyerg What do you think about this?

Reply:
@sriyerg which approach do you prefer? The solution to change the Riviera settings as shown in https://github.com/lowRISC/opentitan/pull/2242#discussion_r427207114 seems like the safest option to me.

Reply:
Is knowing which sub-tool the message is being printed from a useful information to you? If not, then disabling it sounds good to me.

Reply:
I wonder if there is a command line switch to disable this instead. If there is, then you can add it to `build_opts` and `run_opts` instead.